adam/curriculum_to_html.py

```diff
@@ -128,48 +128,33 @@ def _dump_instance_group(
-                "\n</ul>\n<ul>\n<p>Relations</p> "
-                "\n\t<li>These define relations between two objects; e.g. smallerThan(A, B) indicates that object A is "
-                "smaller than object B.</li> "
-                "\n\t<li>IN_REGION is a special relation that holds between a SituationObject and a Region. These "
-                "relations can be nested, and for readability, they do not fall under 'Other Relations'."
-                "\n\t<li>RELATION(OBJECT_0, OBJECT_1)</li>"
-                "\n</ul>\n<ul>\n<p>Arrows</p>"
-                "\n\t<li>A relation preceded by 'Ø --->' is one that begins to exist by the end of the situation.</li>"
-                "\n\t<li>A relation followed by '---> Ø' is one that ceases to exist by the end of the situation.</li>"
-                "\n\t<li>A relation without an arrow holds true through the duration of the situation.</li>"
-                "\n</ul>\n<ul>\n<p>Part-of nesting</p>"
-                "\n\t<li>PART_OF relations indicate that an object is a part of another. Objects that form a "
-                "structural schema can be represented in terms of several part-of relations.</li> "
-                "\n\t<li>e.g. a finger is a part of a hand which is part of an arm which is part of a body</li>"
-                "\n</ul>\n<ul>\n<p>Regions</p> "
-                "\n\t<li>These represent regions of space. They are defined in terms of distance and/or direction with "
-                "respect to a reference object.</li>"
-                "\n\t<li>e.g. the inside of a box may be represented as Region(box, distance=INTERIOR, "
-                "direction=None)</li> "
-                "\n\t<li>Region(REFERENCE_OBJECT, distance=DISTANCE, direction=DIRECTION)</li>"
-                "\n</ul>\n<ul>\n<p>Spatial Paths</p>"
-                "\n\t<li>A SpatialPath specifies the path that some object takes during a situation. These are "
-                "defined in terms of a PathOperator and whether or not the orientation changed, with respect to a "
-                "reference object and an optional reference axis.</li>"
-                "\n\t<li>e.g. the path of a falling object may be represented as SpatialPath(operator=TOWARD, "
-                "reference_object=GROUND, reference_axis=None, orientation_changed=False)</li>"
-                "\n\t<li>SpatialPath(OPERATOR, REFERENCE_OBJECT, REFERENCE_AXIS, ORIENTATION_CHANGED)</li>"
+                "\n\t<li>Part of relations (e.g. a head being a part of a person) are represented with the parts 
+                "\n\t appearing as sub-bullets of an object (e.g. head as a sub-bullet under person).</li>"
+                "\n\t<li>Numeric indices (e.g. person_0 person_1 leg_0, leg_1) distinguish between objects of the"
+                "\n\t same class. </li>"
+                "\n\t<li>Region and in-region relations describe the spatial relations between objects</li>"
```

Comment:
Just `IN_REGION`, I think?

Reply:
@rgabbard what does Region do then?

Reply:
@marjorief `Region` is a class which holds an `object` and `distance` indicators for use in mapping relationships. This allows us to define that a cup is in a proximal relationship to a book. Or example being `in-region(cup_0, REGION(table_0,distance=exterior-but-in-contact))` so that we know the cup is on the exterior region of the table.

Reply:
@marjorief : basically, `IN_REGION` is the relation type; a `Region` object is the second argument of an `IN_REGION` relation.  So if there is a ball on a table, there is the following relation:
```
IN_REGION(ball, Region(reference=table, distance=EXTERIOR_BUT_IN_CONTACT, direction=GRAVITATIONAL_UP)
```